techempower/scripts/tfb-cleanup.sh

```diff
@@ -53,31 +53,37 @@ fi
 # Removes the tfb-qrh instances from openshift
 # output: Removes the tfb-qrh and tfb-database deployments, services, service monitors and routes
 function remove_tfb_openshift() {
-	TFB_DEPLOYMENTS=($(oc get deployments --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
+	TFB_DEPLOYMENTS=($(${K_EXEC} get deployments --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
 
 	for de in "${TFB_DEPLOYMENTS[@]}"
 	do
-		oc delete deployment ${de} --namespace=${NAMESPACE}
+		${K_EXEC} delete deployment ${de} --namespace=${NAMESPACE}
 	done
 
 	#Delete the services and routes if any
-	TFB_SERVICES=($(oc get svc --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
+	TFB_SERVICES=($(${K_EXEC} get svc --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
 	for se in "${TFB_SERVICES[@]}"
 	do
-		oc delete svc ${se} --namespace=${NAMESPACE}
+		${K_EXEC} delete svc ${se} --namespace=${NAMESPACE}
 	done
-	TFB_ROUTES=($(oc get route --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
+	TFB_ROUTES=($(${K_EXEC} get route --namespace=${NAMESPACE} | grep -e "tfb-qrh" -e "tfb-database" | cut -d " " -f1))
```

Comment:
I think route is specific to openshift please add a cluster check here as the script fails with "error: the server doesn't have a resource type "route" on minikube